ee/clickhouse/views/test/test_clickhouse_insights.py

```diff
@@ -28,6 +30,51 @@ def _create_event(**kwargs):
 class ClickhouseTestInsights(
     ClickhouseTestMixin, LicensedTestMixin, insight_test_factory(_create_event, _create_person)  # type: ignore
 ):
+    def test_insight_trends_basic(self):
+        with freeze_time("2012-01-14T03:21:34.000Z"):
+            _create_person(distinct_ids=["1"], team=self.team)
+            _create_person(distinct_ids=["2"], team=self.team)
+            _create_event(team=self.team, event="$pageview", distinct_id="1")
+            _create_event(team=self.team, event="$pageview", distinct_id="2")
+
+        with freeze_time("2012-01-15T04:01:34.000Z"):
+            response = self.client.get(
+                f"/api/projects/{self.team.id}/insights/trend/?events={json.dumps([{'id': '$pageview'}])}"
+            ).json()
+
+        self.assertEqual(response["result"][0]["count"], 2)
+        self.assertEqual(response["result"][0]["action"]["name"], "$pageview")
+
+        self.assertEqual(response["result"][0]["data"][-2], 2)
+
+        with freeze_time("2012-01-15T04:01:34.000Z"):
+            response = self.client.get("/" + response["result"][0]["persons_urls"][-2]["url"]).json()
```

Comment:
I think the url should be absolute already?

Also, I can't wait to make the response not just a bunch of separate arrays. Obvs. not for this PR though.

Reply:
I removed the first slash because on the frontend our api client handles the url best when it doesn't start with the slash

Reply:
right, is there any reason we wouldn't return absolute from the api? aside from what the javascript client is expecting. It seems like an odd thing to do and a potential place for errors to slip in.

Reply:
wait i confused the terminology. We should return absolute urls like in `format_next_url` we build the whole thing. I mirrored the handling from your correlations and forgot to change it

Reply:
Given how our query classes are set up and lack of access to the request object, it seems easier to keep it as relative for now